### PR TITLE
Use correct matrix server list for mainnet

### LINKED ISSUE #2263: Use correct server list for mainnet
## Description

Currently the list of available matrix servers is hard coded to the list of goerli testnet servers.
https://github.com/raiden-network/light-client/blob/c79d70f93b71d9d11c83c1c7403615d43811df7f/raiden-ts/src/config.ts#L116-L117

Depending on the network the list should be adapted.

## Acceptance criteria
- Use the correct server lists for at least goerli and mainnet

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by palango**
**Thank you for submitting this pull request :)**

Fixes #2263 

**Short description**

Use the correct matrix server list on mainnet.

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-ts/src/config.ts [hunk 1]
```diff
@@ -112,15 +112,20 @@ export function makeDefaultConfig(
   { network }: { network: Network },
   overwrites?: PartialRaidenConfig,
 ): RaidenConfig {
+  const networkName = getNetworkName(network);
+  const matrixServerInfos =
+    networkName === `mainnet`
```
Comment (palango) [inline raiden-ts/src/config.ts:117]: @andrevmatos Should we compare using this name, or the `chain_id`?

Comment (andrevmatos) [reply raiden-ts/src/config.ts:117]: On `raiden.ts:1126`, we use `network.chainId !== 1` to check for mainnet.. I think it'd be better than using this string which is just intended to be a _human-readable_ name of it, although it works.

File: raiden-ts/src/config.ts [hunk 2]
```diff
@@ -112,15 +112,20 @@ export function makeDefaultConfig(
...
+    matrixServerLookup: matrixServerInfos,
     settleTimeout: 500,
     revealTimeout: 50,
     httpTimeout: 30e3,
-    discoveryRoom: `raiden_${getNetworkName(network)}_discovery`,
-    pfsRoom: `raiden_${getNetworkName(network)}_path_finding`,
-    monitoringRoom: `raiden_${getNetworkName(network)}_monitoring`,
+    discoveryRoom: `raiden_${networkName}_discovery`,
+    pfsRoom: `raiden_${networkName}_path_finding`,
+    monitoringRoom: `raiden_${networkName}_monitoring`,
```
Comment (andrevmatos) [inline raiden-ts/src/config.ts:128]: Thanks =)

**Review by andrevmatos — APPROVED** [review]
lgtm, merge after you do the chainId check, if you may. thanks

**Pull request merged**